### DISCUSSION TITLE
Resolved: Browsers ignoring generated CSS styles

### DISCUSSION THREAD

**Discussion by eugis42** [tailwindlabs/tailwindcss#6929] [Help]
Edit: resolved

Tailwind-CLI 3.0.11

Tailwind generated css is fine, browser and server side cache disabled, when I check the css through the browser the classes are there - the browser just ignores them. I tried multiple browsers, deleting cache. I feel like this is something really simple and stupid please help :D

**Comment by zeconslab** [comment]
como lo resolviste? por que en mi caso las clasesa no funcionan al menos las de color
